test/remote/gateways/remote_d_local_test.rb

```diff
@@ -50,6 +50,14 @@ def test_successful_purchase
     assert_match 'The payment was paid', response.message
   end
 
+  def test_successful_purchase_with_save_option
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(save: true))
+    assert_success response
+    assert_equal true, response.params['card']['save']
+    assert_not_empty response.params['card']['card_id']
```

Comment:
Can you also assert that we're receiving the `[card][type]` in the response, since that's one of the motivations for adding this?

Reply:
Hey @jcreiff after a little more digging, I see that the card['type'] is not returned in the sandbox tests response even when calling verify instead of purchase. This may be a limitation of the gateway sandbox but if there's something I'm missing, please let me know, thanks!

Reply:
hey @jcreiff the sandbox is now returning the card type so i added the assertion to the remote test, thanks!